diff -Naur rasdaemon-0.5.6/mce-intel-p4-p6.c rasdaemon-0.5.6.fixes/mce-intel-p4-p6.c
--- rasdaemon-0.5.6/mce-intel-p4-p6.c	2015-09-11 14:55:59.000000000 -0600
+++ rasdaemon-0.5.6.fixes/mce-intel-p4-p6.c	2015-09-11 15:10:26.175183310 -0600
@@ -127,7 +127,7 @@
 
 	for (i = 0; i < ARRAY_SIZE(p4_model); i++) {
 		if (model & (1 << p4_model[i].value))
-			mce_snprintf(e->error_msg, p4_model[i].str);
+			mce_snprintf(e->error_msg, "%s", p4_model[i].str);
 	}
 }
 
